charts/spire/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: spire
 description: >
   A Helm chart for deploying the complete Spire stack including: spire-server, spire-agent, spiffe-csi-driver, spiffe-oidc-discovery-provider and spire-controller-manager.
 type: application
-version: 0.5.1
+version: 0.5.2
```

Comment:
Is the version bump intentional?

Reply:
So that we can see the test actually works. Cant do that without a dummy commit to the chart. Once verified, we will remove it before merge.

Reply:
can someone take note that the tests are passing, and if there are no more changes other then removing the version bump, (ready for approval), I'll remove it and we can get it merged.